src/v/test_utils/runfiles.cc

```diff
@@ -17,8 +17,13 @@ get_runfile_path([[maybe_unused]] std::string_view path) {
 #ifdef BAZEL_TEST
     using bazel::tools::cpp::runfiles::Runfiles;
     std::string error;
```

Comment:
`error` is set but unused after?

Reply:
If runfiles is null it's used as the error message in the `std::runtime_error`